lines.css

```diff
@@ -138,4 +138,17 @@ body {
     color: black;
     font-size: 12px;
     font-family: "Montserrat", sans-serif;
+}
+
+.permament-tools {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+.history-button {
+    height: 25px;
```

Comment:
Size of the image `448x512` - not square, but you use `25x25`. I would use button with image inside it. Specify only width (or only height) for the image and align it to center by vertical and horizontal